packages/ui/src/forum/components/threads/ThreadItem.tsx

```diff
@@ -18,21 +20,21 @@ interface ThreadAnswerProps {
 }
 
 export interface ThreadItemContentProps {
-  title: string
-  date?: string
-  content?: string
+  thread: ForumThread
   badges?: ThreadBadgeProps[]
   answers?: ThreadAnswerProps[]
   halfSize?: boolean
   empty?: boolean
 }
 
-export const ThreadItem = ({ title, date, content, badges, answers, halfSize, empty }: ThreadItemContentProps) => {
+export const ThreadItem = ({ thread, badges, answers, halfSize, empty }: ThreadItemContentProps) => {
+  const originalPost = useThreadOriginalPost(thread.id)
```

Comment:
There's a weird delay while browsing the threads. I'm wondering what we could do about that.

The simplest solution would be to delay loading title+text until we have all the data.